_posts/2020-11-05-markdown-tutorial.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Markdown Tutorial"
+date:   2020-11-06 12:54:04 -0700
+categories: markdown tutorial
+---
+In this post, I will be writing a tutorial on how to create a Markdown from scratch and all that a beginner needs to know about it.
+
+Markdown is a lightweight Markup language with easy to use plain text formatting syntax that was created by John Gruber and Aaron Swartz in 2004.
+It is used for styling different forms of writing like formatting README files, writing messages in online discussions platform, writing content for the web and to create rich text.
+
+## Basic Markdown Syntaxes
+Syntaxes are set of rules for grammar and spelling that defines the combination of symbols considered to be correctly structured in a particular programming language.
+
+##### Headings
+Headings in Markdown are any line which is prefixed with the hashtag (#) symbol. The number of hashtags represents the heading level. A total of 6 levels can be made with 6 hashtags but in most writings, only 3 is commonly used. The syntax in block quotes are showing what the text looks like in Markdown.
+
+```
+# Heading 1
+## Heading 2  
+### Heading 3
+```
+# Heading 1
+## Heading 2  
+### Heading 3
+
+##### Texts
+When writing text in Markdown, you can place emphasis on a word by wrapping it in asterisks (*) or underscores (_). They can range from 1 to 3 symbols based on the extent of the emphasis. The syntax in block quotes are showing what the text looks like in Markdown.
+```
+*Item 1*  
+**Item 2**  
+__Item 3__
+```
+*Item 1*  
+**Item 2**  
+__Item 3__
+
+When adding a link to a text, the text will we wrapped with the links in square brackets [] followed by the URL to be linked in brackets (). The syntax in block quote is showing what the text looks like in Markdown.
+```
+[Atom](https://atom.io/)
+```
```

Comment:
Not sure if this is a real problem but I don't think you really need to have the same code repeated again outside of the chunk.

Reply:
@Alicebada I think its fine to leave this, because the following code shows the image. I think Anthony was mis-interpretting what the following code did.